includes/Admin/Editor.php

```diff
@@ -479,4 +480,93 @@ protected function setup_lock( int $story_id ): void {
 			wp_set_post_lock( $story_id );
 		}
 	}
+
+	/**
+	 * Filter the editor_max_image_size.
```

Comment:
The doc block here absolutely should mention that this is an exact copy of the core function with certain tweaks.